frontend/src/components/AdminApplicationOverview/index.tsx

```diff
@@ -30,11 +30,10 @@ class _Container extends React.Component<Props> {
         applications={this.props.applications}
       />);
   }
-
 }
 
 const mapStateToProps = (state: IStore) => ({
-  applications: state.applications.applications,
+  applications: state.adminApplicationList.applications,
```

Comment:
Why split applications into application and adminApplicationList  instead of both being under application?